lib/reporters/verbose.js

```diff
@@ -3,6 +3,7 @@ var prettyMs = require('pretty-ms');
 var figures = require('figures');
 var chalk = require('chalk');
 var plur = require('plur');
+var repeatString = require('repeat-string');
```

Comment:
Can you use https://github.com/sindresorhus/repeating? We already depend on modules using it, so it would mean no extra dependency.